### PR TITLE
added depth and contrast to hero text

### PR DESCRIPTION
**Pull request by vinyll**
### Before

screen shot 2018-08-22 at 14 54 22

### After 

screen shot 2018-08-22 at 14 54 10

### CODE REVIEW

**Comment by fuentesloic** [comment]
Nice (de Brice) 🤙

File: content/styles/03-components/hero.css [hunk 1]
```diff
@@ -10,6 +10,7 @@
   & h2,
   & h3 {
     color: var(--white);
+    text-shadow: 0 0 .3rem #555;
```
Comment (fuentesloic) [inline content/styles/03-components/hero.css:13]: We have a var for gray `--gray: #333;` do you think you could use it to replace `#555` ? 'cause they are pretty close. If not no problem ^^ or maybe change `--gray: #333;` for `--gray: #555;`

Comment (fuentesloic) [reply content/styles/03-components/hero.css:13]: other possible case we have an `--shadow-color: rgba(50, 50, 50, .15);` we could consider a `--text-shadow: 0 0 .3rem #555;` then `text-shadow: var(--text-shadow);` In this case that could be re use (by me for exemple ^^)

Comment (vinyll) [reply content/styles/03-components/hero.css:13]: We could, but the result is different. I tried from `#999` to `#222` and `#555` seemed to be the most appropriate to me.
I'd appreciate a different eye tho.

Comment (fuentesloic) [reply content/styles/03-components/hero.css:13]: and about `text-shadow: var(--text-shadow);` ??

Comment (vinyll) [reply content/styles/03-components/hero.css:13]: At the same time I'm not sure if it's interesting to reuse this part.
The _hero_ banner is so specific that the shadow is contextual to the background image and the size of the text.
Therefore I wouldn't bother with aligning to other variables that are useful in a different context.

Comment (fuentesloic) [reply content/styles/03-components/hero.css:13]: totally true!

**Review by fuentesloic — APPROVED** [review]

**Pull request merged**